recipes/netpbm/config.mk

```diff
@@ -0,0 +1,77 @@
+DEFAULT_TARGET = nonmerge
+BUILD_FIASCO = Y
+LD = $(CC)
+LINKERISCOMPILER=Y
+LINKER_CAN_DO_EXPLICIT_LIBRARY=Y
+INTTYPES_H = <inttypes.h>
+HAVE_INT64 = Y
+WANT_SSE = N
+CC_FOR_BUILD = $(CC)
+LD_FOR_BUILD = $(LD)
+CFLAGS_FOR_BUILD = $(CFLAGS_CONFIG)
+LDFLAGS_FOR_BUILD = $(LDFLAGS)
+WINDRES = windres
+INSTALL = $(SRCDIR)/buildtools/install.sh
+STRIPFLAG = -s
+SYMLINK = ln -s
+MANPAGE_FORMAT = nroff
+AR = ar
+RANLIB = ranlib
+LEX = flex
+PKG_CONFIG = pkg-config
+CFLAGS = -L${PREFIX}/lib -I${PREFIX}/include -O3 -ffast-math  -pedantic -fno-common -Wall -Wno-uninitialized -Wmissing-declarations -Wimplicit -Wwrite-strings -Wmissing-prototypes -Wundef
```

Comment:
You should append these to the ones from the environment

Reply:
Ok, thanks!

Reply:
That's the issue. conda-build environment sets the value `${CFLAGS}` and you shouldn't override it like this. You should append to that

Reply:
Seems that changing it to append (`+=`) do not cause the problems I imagined. So I will change it in the branch.